src/main/java/org/corfudb/cmdlets/corfu_smrobject.java

```diff
@@ -175,7 +186,19 @@
                     // caused by a disconnection with the remote endpoint.  That kind
                     // of non-determinism is evil.  Just retry a few times via 'for' loop.
                     log.warn("WHOA, 'Disconnected endpoint', looping...\n");
-                    try { Thread.sleep(50); } catch (InterruptedException ie){};
```

Comment:
I know this isn't part of this PR, but line 177 (above) inserts an informal log message which doesn't even tell the user which endpoint disconnected.